Podfile

```diff
@@ -7,6 +7,7 @@ inhibit_all_warnings!
 target :Awful do
   pod 'AFNetworking', '2.5.0'
   pod 'ARChromeActivity', '1.0.4'
+  pod 'AsyncDisplayKit', '1.1.1'
```

Comment:
The next version which adds the editable text node is 1.2, which I think follows semver. Semver tends to get murky the more interesting or complex a project, so I don't sweat it too much and just specify MAJOR.MINOR.

Then again, for stuff like node.js modules, I do specify versions fully, because like you I don't trust maintainers. Same goes for maven when doing Java (despite many responsible maintainers), because of the huge size of the ecosystem (and also perfectly reproducible builds).